### PR TITLE
[sl] add Slovenian sentences, responses

### PR DESCRIPTION
**Pull request by andrejs2**

### CODE REVIEW

File: sentences/sl/light_HassLightSet.yaml [hunk 1]
```diff
@@ -2,4 +2,38 @@ language: sl
...
+      #brightness min/max
+      - sentences:
+          - "<spremeni> <name> na {brightness_level:brightness}"
+          - "<spremeni> [jakost|moč|intenzivnost|svetlost] [svetila|razsvetljave|osvetlitve|svetilke|luči] [v|na] <name> na {brightness_level:brightness} [moči|jakosti|svetlosti|odstotkov|procentov]"
+        slots:
+          brightness: 100
+        response: brightness
+
+      - sentences:
+          - "<spremeni> [jakost|moč|intenzivnost|svetlost] [svetila|razsvetljave|osvetlitve|svetilke|luči] [v|na] <area> [na] {brightness_level:brightness} [moči|jakosti|svetlosti|odstotkov|procentov]"
```
Comment (spuljko) [inline sentences/sl/light_HassLightSet.yaml:23]: This shouldn't be optional [jakost|moč|intenzivnost|svetlost] [svetila|razsvetljave|osvetlitve|svetilke|luči] - change to round brackets

File: sentences/sl/light_HassTurnOff.yaml [hunk 2]
```diff
@@ -2,7 +2,15 @@ language: sl
 intents:
   HassTurnOff:
     data:
-      - sentences: []
+      - sentences:
+          - "<izključi> [luč[i|ke|ko|ki]|svetilo|razsvetljavo|svetilko] [v|na] [sobi|prostoru] <area>"
```
Comment (spuljko) [inline sentences/sl/light_HassTurnOff.yaml:6]: Change to round brackets [luč[i|ke|ko|ki]|svetilo|razsvetljavo|svetilko]

File: sentences/sl/light_HassLightSet.yaml (hunk 1 above)
Comment (andrejs2) [reply sentences/sl/light_HassLightSet.yaml:23]: Changed line 23 - in brackets. Thanks!

File: sentences/sl/light_HassTurnOff.yaml (hunk 2 above)
Comment (spuljko) [reply sentences/sl/light_HassTurnOff.yaml]: You should do it in all places where sentence like "action domain state" is used. Action is turn on-off/increase-decrecrese brighhtnes etc. Domain is ligh/lamp etc. State id on/off, pecetntage etc.  So, all 3 parts are needed in (). Anything in between is optional un [].

Comment (andrejs2) [reply sentences/sl/light_HassTurnOff.yaml]: Thanks @spuljko , I'm working on it now. Had to translate all tests folder in order to run tests...I see there are still sentences to be fixed.

**Comment by tetele** [comment]
@andrejs2 with Git, you can stage multiple files and then push a single commit https://www.nobledesktop.com/learn/git/stage-commit-files

In some repos, such as this one, each commit has some actions attached to it (e.g. starting a verification pipeline) so committing files individually triggers that pipeline over and over. Not to mention there will be a flood of notification emails for the repo maintainers.

**Comment by andrejs2** [comment]
I made a mess, sorry. I'l try to clean up and close thiss PR. Opened new PR for lights which btw are fixed now. Tests should pass :)
Thanks for the link, I was to excited to push through that I copied main files to my main branch and then made PR...ahhhh

**Pull request closed without merging**